src/Microsoft.Crank.RegressionBot/Program.cs

```diff
@@ -297,6 +299,11 @@ private static async Task<string> CreateIssueBody(IEnumerable<Regression> regres
 
                 body += regressionBlock;
 
+                if (body.Length > 65536)
+                {
+                    throw new Exception($"Body too long ({body.Length} > 65536 chars)");
```

Comment:
Any way to try truncating first? I assume this error shows up in the regression bot AzDo run? Does it send an email if it fails?

Reply:
Can truncate because it's a binary payload for serialized data, so couldn't be deserialized and the issue would not be tracked (the issues would be recreated every time they are detected).

I need to create another PR that will actually send a != 0 exit code such that Azdo knows the job is failing. It's currently green. Need to check how to handle that with System.CommandLine

Reply:
Ideally issues would be filled with regressions until there is no more space. And have a block of MSGPACK per issue then create a new page, the idea being to limit the number of GH issues created.

I was surprised of the final size of the serialized text. Maybe Base64 is not optimal here. But even from MsgPack the size is significant, and I checked only the required data is serialized.